pkg/apis/openebs.io/upgrade/v1alpha1/upgraderesult.go

```diff
@@ -130,6 +135,8 @@ type ResourceDetails struct {
 	APIVersion string `json:"apiVersion"`
 	// Namespace of the resource
 	Namespace string `json:"namespace"`
+	// Generation of resource
```

Comment:
The comment is not clear. Can we explain further.

Reply:
done